my-sms/app/views/students/show.html.erb

```diff
@@ -1,40 +1,31 @@
-<p id="notice"><%= notice %></p>
-
-<p>
-  <b>Title:</b>
-  <%= @student.title %>
-</p>
-
-<p>
-  <b>First name:</b>
-  <%= @student.first_name %>
-</p>
-
-<p>
-  <b>Middle name:</b>
-  <%= @student.middle_name %>
-</p>
-
-<p>
-  <b>Last name:</b>
-  <%= @student.last_name %>
-</p>
-
-<p>
-  <b>Email:</b>
-  <%= @student.email %>
-</p>
-
-<p>
-  <b>Birth date:</b>
-  <%= @student.birth_date %>
-</p>
-
-<p>
-  <b>Gender:</b>
-  <%= @student.gender %>
-</p>
-
-
-<%= link_to 'Edit', edit_student_path(@student) %> |
-<%= link_to 'Back', students_path %>
+<%- model_class = Student -%>
+<div class="page-header">
+  <h1><%=t '.title', :default => model_class.model_name.human.titleize %></h1>
+</div>
+
+<dl class="dl-horizontal">
+  <dt><strong><%= model_class.human_attribute_name(:title) %>:</strong></dt>
+  <dd><%= @student.title %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:first_name) %>:</strong></dt>
+  <dd><%= @student.first_name %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:middle_name) %>:</strong></dt>
+  <dd><%= @student.middle_name %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:last_name) %>:</strong></dt>
+  <dd><%= @student.last_name %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:email) %>:</strong></dt>
+  <dd><%= @student.email %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:birth_date) %>:</strong></dt>
+  <dd><%= @student.birth_date %></dd>
+  <dt><strong><%= model_class.human_attribute_name(:gender) %>:</strong></dt>
+  <dd><%= @student.gender %></dd>
+</dl>
```

Comment:
I like the use of `dl`, `dt` and `dd` here for semantic html and accessibility. I think the <strong>'s could probably be replaced with a `<span class="your-class">` and then some css to style that appropriately (ie. make it bold).

Reply:
Add some custom classes in 7d8ac2e028d62a8a9cfb09bbec0907d51adc1c76
Didn't really end up changing much over the default bootstrap CSS applied to dl/dt/dd elements but was a lesson in the asset pipeline basics.